docs/en/repository.md

```diff
@@ -53,6 +53,12 @@ Not anymore! Kdyby adds possibility to search by relation properties.
 $commentsRepository->findBy(['article.title' => $title]);
 ```
 
+Using `like` is also possible:
+
+```php
+$commentsRepository->findBy(['article.title like' => '%' . $title . '%']);
```

Comment:
add like to the listed operations just below instead of here

Reply:
I was deciding between the two and chose the wrong one :)

Hopefully I've updated the PR correctly.